packages/extras-buildpkgs/htop/debian/patches/series

```diff
@@ -5,4 +5,5 @@ remove-fancy-flashing.patch
 fix-cpufreq-meter-on-biglittle.patch
 fix-cpufreq-meter-for-non-root.patch
 cpu-temperatute-from-hwmon.patch
+fix-unchecked-buffer-writes.patch
```

Comment:
IMO not ideal to fix this in an additional patch file instead of the original patch file that introduces the issues. In particular because there's little reference in the list that indicates that this patch is important when using the `avafinger` patch. Ordering this patch directly after that patch and renaming to something that clearly convenes that this patch bugfixes the previous patch (e.g. `fix-unchecked-buffer-writes-in-avafinger-cpu-monitor.patch`) is probably preferable.

Reply:
Also a good point.
I will make it more obvious that it fixes the issues introduced in avafinger's patch.

I still want the fixes separated from the main patch though as it simply represents a squash of the changes from @avafinger's [repo](https://github.com/avafinger/htop_2.2.2) and it makes it easier to incorporate the "midstream" ;-) changes in the future.

Maybe even this fix itself will get there one day...